scripts/process_simple_articles.py

```
@@ -0,0 +1,155 @@
+from src.workflow_config import get_workflow_config
+from datetime import date
+
+
+def process_simple_articles():
+    """
+    Process articles with simple relevance filtering.
+    This replaces the complex processing pipeline with a streamlined approach.
+    """
+    print("🧠 Starting simple LLM processing (relevance filtering only)...")
+    print(f"Using LLM provider: {os.getenv('LLM_PROVIDER', 'openrouter')}")
+    print(f"📊 Processing articles for relevance (no complex IOC extraction)")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)